include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/Solver/IDESolver.h

```diff
@@ -54,37 +36,29 @@
 #include "phasar/Utils/PAMMMacros.h"
 #include "phasar/Utils/Table.h"
 
-namespace psr {
-
-// Forward declare the Transformation
-template <typename AnalysisDomainTy, typename Container>
-class IFDSToIDETabulationProblem;
+#include "nlohmann/json.hpp"
 
-struct NoIFDSExtension {};
+#include "boost/algorithm/string/trim.hpp"
```

Comment:
Maybe on the fly get rid of `boost::trim` and replace it with `llvm::StringRef::trim`?